test/component/scheduler/perf/test-performance.sh

```diff
@@ -34,13 +37,10 @@ trap cleanup EXIT
 kube::etcd::start
 kube::log::status "performance test start"
 
-# TODO: set log-dir and prof output dir.
-DIR_BASENAME=$(basename `pwd`)
-go test -c -o "${DIR_BASENAME}.test"
 # We are using the benchmark suite to do profiling. Because it only runs a few pods and
 # theoretically it has less variance.
-"./${DIR_BASENAME}.test" -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -logtostderr=false
+go test -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -logtostderr=false
```

Comment:
If we run normal benchmarking, we will do:
```
go test -run XXX -bench.
```

But if we want to do profiling, we need to
```
# compile it
go test -c # generate xxx.test
# benchmark and generate profile file
./xxx.test -test.run=xxx -test.bench=. -testcpuprofile=prof.out
# analyze profile output
go tool pprof xxx.test prof.out
```
The last step needs both binary and "prof.out".